bika/lims/browser/stickers.py

```diff
@@ -149,7 +159,8 @@ def _populateItems(self, item):
             return [[None, None, item]]
         items = []
         for part in parts:
-            items.append([ar, sample, part])
+            for copy in range(self.copies_count):
+                items.append([ar, sample, part])
```

Comment:
Note that in this loop we are dealing with partitions (e.g. a Sample can have more than one partition and we ensure here we print a sticker for every single partition) and is never called when the object is a Worksheet or a Batch for instance (see L1575 and L159).

Add this logic into a function `_resolve_number_of_copies` and call it before L95 https://github.com/senaite/senaite.core/pull/618/files#diff-288974376469bb5c86e96399ce16a703R94